src/spork.cpp

```diff
@@ -2,18 +2,17 @@
 // Distributed under the MIT software license, see the accompanying
 // file COPYING or http://www.opensource.org/licenses/mit-license.php.
 
+#include "spork.h"
+
 #include "chainparams.h"
 #include "validation.h"
 #include "messagesigner.h"
 #include "net_processing.h"
 #include "netmessagemaker.h"
-#include "spork.h"
+#include "base58.h"
```

Comment:
nit: move new include to the top of this group of includes to keep alphabetical order